### ISSUE TITLE
When a draft is set for future publication, and then you re-edit it, hydrate the publication date/time automatically

### ISSUE THREAD

**Issue by CrispinF** [cloudscribe/cloudscribe.SimpleContent#320]
As things stand I have to enter this again or lose it:
![image](https://user-images.githubusercontent.com/10574730/43308129-61f34724-9178-11e8-8937-bb4d48b81f06.png)

I think that given some effort has gone into setting this future date/time, opening the draft for re-editing should automatically put this value back into the date/time control.

**Comment by joeaudette** [comment]
If you schedule a draft to publish later the future publication date is not lost if you "Save Draft" again. There is only ever one draft so edits to the draft before publishing will be included once the draft is published.
I think it will be cleaner and less complicated to just update the message so it is clear that you can save draft and the new changes will be included when published, rather than conditionally populating the datepicker which would be more complicated.

**Comment by CrispinF** [comment]
@joeaudette you're right. I mistakenly thought the future date/time was reset.  So I agree with your suggestion.

**Comment by joeaudette** [comment]
this is updated with a better message in the latest nugets
